docs/content/latest/integrations/kairosdb.md

```diff
@@ -73,32 +70,139 @@ Use HELP for help.
 ycqlsh>
 ```
 
-- Run the following YCQL commands to verify it is working:
+- Run the following YCQL commands, and connect to the `kairosdb` keyspace to verify it is working:
 
-```sql
+```cql
 ycqlsh> describe keyspaces;
 ```
 
 ```output
 kairosdb  system_schema  system_auth  system
 ```
 
-```sql
+```cql
 ycqlsh> use kairosdb;
+```
+
+```cql
 ycqlsh:kairosdb> describe tables;
 ```
 
 ```output
-row_keys       data_points    string_index
-row_key_index  service_index  row_key_time_index
+tag_indexed_row_keys  row_key_index  service_index  row_key_time_index
+row_keys              data_points    string_index   spec
 ```
 
-## 5. Test KairosDB
+## Test KairosDB
+
+- Start Postman and create a new workspace.
 
 ### Push data
 
-Push metric data into KairosDB as per the instructions [here](https://kairosdb.github.io/docs/PushingData.html).
+- Select the POST API request and add the following URL in the `Enter request URL` box:
```

Comment:
"Select the POST API request" - can you be more specific? Do you click some button, or other UI object?

Reply:
Added more steps and screenshots to be more clear.